code/game/objects/items.dm

```diff
@@ -1066,3 +1066,65 @@ var/global/list/image/blood_overlays = list()
 
 /obj/item/animationBolt(var/mob/firer)
 	new /mob/living/simple_animal/hostile/mimic/copy(loc, src, firer, duration=SPELL_ANIMATION_TTL)
+
+/obj/item/proc/is_worn(mob/user)
+	var/mob/living/carbon/monkey/M
+	var/mob/living/carbon/human/H
+	if(istype(user, /mob/living/carbon/monkey))
+		M = user
+	if(istype(user, /mob/living/carbon/human))
+		H = user
+	if(!(H || M))
+		return 0
+	if(H)
+		if(slot_flags & SLOT_OCLOTHING)
```

Comment:
@PJB3005 Having a 0 to 15 loop for every clothing object on every human on every tick sounds like not the best idea